test/bus_test.dart

```diff
@@ -13,6 +13,61 @@ import 'package:rohd/src/exceptions/logic/logic_exceptions.dart';
 import 'package:rohd/src/utilities/simcompare.dart';
 import 'package:test/test.dart';
 
+class LogicIsInModule extends Module {
```

Comment:
These tests look great!

Since the file is already so large, why don't we move these into a new file speficially for testing `isIn`?